pkg/api/validation/validation.go

```diff
@@ -3692,7 +3692,8 @@ func ValidateSecurityContext(sc *api.SecurityContext, fldPath *field.Path) field
 
 	if sc.Privileged != nil {
 		if *sc.Privileged && !capabilities.Get().AllowPrivileged {
-			allErrs = append(allErrs, field.Forbidden(fldPath.Child("privileged"), "disallowed by policy"))
+			allErrs = append(allErrs, field.Forbidden(fldPath.Child("privileged"),
+				"privileged containers are disallowed on this cluster by a system administrator"))
```

Comment:
I don't mind to modify the message but  the main intention of this change was to remove word "policy" from the message. When I saw this I thought about PodSecurityPolicy but this is completely unrelated. Ideally it should tell that administrator didn't pass `--allow-privileged` option to kubelet and api-server but probably you wouldn't agree with such message too?